moped-editor/src/components/DataGridPro/DataGridActions.js

```diff
@@ -58,6 +62,7 @@ const DataGridActions = ({
       <GridActionsCellItem
         icon={<CheckIcon sx={defaultEditColumnIconStyle} />}
         label="Save"
+        key="save"
```

Comment:
DataGrid expects an array of components for columns with `actions` type in the column config. I noticed it wasn't adding keys to this abstracted component the way it would if these components were defined directly in the config which is odd. It was causing console warnings, so, I added keys based on the labels since these components never change order in the component tree.

Reply:
+1 nice catch